docs/sdks/uid2-sdk-ref-csharp-dotnet.md

```diff
@@ -107,60 +102,113 @@ Available information returned through the SDK is outlined in the following tabl
 | `KeysNotSynced` | The client has failed to synchronize keys from the UID2 service. |
 | `VersionNotSupported` |  The client library does not support the version of the encrypted token. |
 
+### Encryption Response Content
+
+When encrypting with the `SharingClient`, the SDK will return the following information:
+
+| Property | Description |
+| :--- | :--- |
+| `Status` | The encryption result status. For a list of possible values and definitions, see [Encryption Response Statuses](#encryption-response-statuses). |
+| `EncryptedData` | The encrypted UID2 advertising token. |
+
+### Encryption Response Statuses
+
+| Value | Description |
+| :--- | :--- |
+| `Success` | The raw UID2 was successfully encrypted and a UID2 advertising token was returned. |
+| `NotAuthorizedForKey` | The requester does not have authorization to use the encryption key. |
```

Comment:
Feels like the descriptions are just verbose ways of re-stating the enum value without adding useful info. 
Eg `NotAuthorizedForKey` in a sharing context can mean the other participant hasn't given you permission.
`KeysNotSynced` can mean Refresh() hasn't been called or it failed.

That's why I much prefer this reference information to be centralized, where it can be given more love

Reply:
Do you not want me to document these statuses, or are there ways we can improve the descriptions?

These statuses have no descriptions in the code so I reused existing descriptions from the decryption statuses where possible, or had to come up with a description otherwise.

How would we centralize this information?

Reply:
I'd prefer that we add these 1 line comments to https://github.com/IABTechLab/uid2-client-net/blob/main/src/UID2.Client/DecryptionStatus.cs#L3 rather than have a table here.

In terms of centralizing, we could have a page which gives more explanation on what each value could mean and what the consumer is meant to do about it. 

Not necessary for this item though - which is about the change from Uid2Client to BidstreamClient